forge-gui/res/cardsfolder/m/maze_of_ith.txt

```diff
@@ -1,6 +1,8 @@
 Name:Maze of Ith
 ManaCost:no cost
 Types:Land
-A:AB$ Untap | Cost$ T | ValidTgts$ Creature.attacking | TgtPrompt$ Select target attacking creature | SubAbility$ DBPump | IsCurse$ True | StackDescription$ Untap {c:Targeted}. | SpellDescription$ Untap target attacking creature. Prevent all combat damage that would be dealt to and dealt by that creature this turn.
-SVar:DBPump:DB$ Pump | Defined$ Targeted | KW$ Prevent all combat damage that would be dealt to and dealt by CARDNAME. | StackDescription$ Prevent all combat damage that would be dealt to and dealt by that creature this turn.
+A:AB$ Untap | Cost$ T | ValidTgts$ Creature.attacking | TgtPrompt$ Select target attacking creature | SubAbility$ DBPump | SpellDescription$ Untap target attacking creature. Prevent all combat damage that would be dealt to and dealt by that creature this turn.
```

Comment:
I think remove `Prevent all combat damage that would be dealt to and dealt by that creature this turn.` from this SpellDescription$ and add it as SpellDescription in line 5